components/collaboration/apply-now/ApplyNowContent.tsx

```
@@ -0,0 +1,203 @@
+import React from 'react';
+import { ScrollView } from 'react-native';
+import { Card, HelperText, IconButton, List, Paragraph, TextInput } from 'react-native-paper';
+
+interface IProps {
+    note: string,
+    quotation: number | undefined,
+    questions: string[]
+    answers: { [key: string]: string }
+    attachmentLength: number,
+
+    setNote: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)